Dockerfile

```diff
@@ -82,7 +89,7 @@ ARG MTCONNECT_DIR=/etc/mtconnect
 ENV MTCONNECT_DIR=$MTCONNECT_DIR
 
 # install ruby for simulator
-RUN apt-get update && apt-get install -y ruby zip
+RUN apt-get clean && apt-get update && apt-get install -y ruby zip
```

Comment:
`apt-get clean` is redundant, see the [Best practices for writing Dockerfiles](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#apt-get).

> Official Debian and Ubuntu images [automatically run apt-get clean](https://github.com/moby/moby/blob/03e2923e42446dbb830c654d0eec323a0b4ef02a/contrib/mkimage/debootstrap#L82-L105), so explicit invocation is not required.